src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -1633,7 +1637,130 @@ Here's a sample configuration:
 </table>
 
              </TabsPageItem>
+        <TabsPageItem id="maui">
+
+The .NET MAUI agent SDK allows you to configure default settings to change the behavior of the agent. 
+
+If used, make sure to add the configuration flags just before calling `appToken`. 
+
+Here's a sample configuration:
+
+```csharp
+```
```

Comment:
@ndesai-newrelic Can you provide a .NET MAUI sample configuration (similar to the other hybrid agents)?

Reply:
```suggestion
Here's a sample configuration:

```csharp
 using NewRelic.MAUI.Plugin;
            ...
    public App()
        {
            InitializeComponent();

            MainPage = new AppShell();

            CrossNewRelic.Current.HandleUncaughtException();
            CrossNewRelic.Current.TrackShellNavigatedEvents();

            // Set optional agent configuration
            // Options are: crashReportingEnabled, loggingEnabled, logLevel, collectorAddress, crashCollectorAddress
            AgentStartConfiguration agentConfig = new AgentStartConfiguration(true, true, LogLevel.INFO, "mobile-collector.newrelic.com", "mobile-crash.newrelic.com");

            if (DeviceInfo.Current.Platform == DevicePlatform.Android)
            {
                // CrossNewRelic.Current.Start("<APP-TOKEN-HERE>");
                // Start with optional agent configuration 
                 CrossNewRelic.Current.Start("<APP-TOKEN-HERE", agentConfig);
            }
            else if (DeviceInfo.Current.Platform == DevicePlatform.iOS)
            {
                // CrossNewRelic.Current.Start("<APP-TOKEN-HERE>");
                // Start with optional agent configuration 
                 CrossNewRelic.Current.Start("<APP-TOKEN-HERE", agentConfig);
            }
        }



    }
```